client/state/ui/actions.js

```diff
@@ -41,3 +42,10 @@ export function setSection( section, options = {} ) {
 	options.hasSidebar = ( options.hasSidebar === false ) ? false : true;
 	return options;
 }
+
+export function setRecentlySelectedSite( siteId ) {
```

Comment:
Do we need a separate action for this? Can we not have the reducer use the `SELECTED_SITE_SET` action in aggregating the set? Aside: there was some effort previously in renaming these functions to reflect that they accepted an ID, not an object. Should we do the same here? i.e. `setRecentlySelectedSiteId`